portfolio/src/main/webapp/css/custom.css

```diff
@@ -355,3 +355,9 @@ body {
 .dropdown:hover .dropbtn {
   background-color: #6ab5cc;
 }
+
+.rounded-circle {
+  border-radius: 50% !important;
```

Comment:
Avoid using !important if you can. Why do we need it here? What is overwriting the border-radius?

Reply:
There is no need for it in our custom-script, so I have removed it.